packages/mui-codemod/src/deprecations/badge-props/test-cases/excepted.js

```diff
@@ -0,0 +1,24 @@
+import { Badge } from '@mui/material';
+
+<Badge
+  slots={{
+    root: ComponentsRoot,
+  }}
+  slotProps={{ root: componentsRootProps }}
+/>;
+
+<Badge
+  slots={{
+    root: SlotsRoot,
+    badge: ComponentsBadge,
+  }}
+  slotProps={{
+    root: slotsRootProps,
+    badge: componentsBadgeProps,
+  }}
+/>;
+
+<Badge
+  slots={{ root: SlotsRoot, badge: SlotsBadge }}
+  slotProps={{ root: slotsRootProps, badge: slotsBadgeProps }}
```

Comment:
We will have to update this after https://github.com/mui/material-ui/pull/41323 was merged. For this, we need to merge master into this branch:

1. `git checkout master`
2. `git pull upstream master`
3. `git checkout manoj/badge-v6-deprecations`
4. `git merge master`
5. Run the codemod on the `actual.js` file and copy it here

Let me know if you need help 😊

Reply:
i tried running actual.js first.
and used the command  that is given in contributing.md which is 
```node packages/mui-codemod/src/deprecations/badge-props/test-cases/actual.js ```
but got an error stating " Can't use import statement outside a module ".
is there any command to run actual.js?